recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,2 @@
 MACOSX_DEPLOYMENT_TARGET:  # [osx]
-  - '10.12'                # [osx]
+  - '11.0'                # [osx]
```

Comment:
? This means that qt will run only on >=11.0

Reply:
We had strange errors without this. Your help in addressing them would be appreciated!

Reply:
Please read the docs on https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks